tck/features/expressions/map/Map4.feature

```diff
@@ -35,18 +35,28 @@ Feature: Map4 - Field existence check
     When executing query:
       """
       WITH <map> AS map
-      RETURN exists(map.name) AS result
+      RETURN exists(map.<key>) AS result
       """
     Then the result should be, in any order:
       | result   |
       | <result> |
     And no side effects
 
     Examples:
-      | map                             | result |
-      | {name: 'Mats', name2: 'Pontus'} | true   |
-      | {name: null}                    | false  |
-      | {notName: 0, notName2: null}    | false  |
+      | map                                 | key     | result |
+      | {name: 'Mats', name2: 'Pontus'}     | name    | true   |
+      | {name: 'Mats', name2: 'Pontus'}     | name2   | true   |
+      | {name: null}                        | name    | true   |
+      | {name: null, name2: 'Pontus'}       | name    | true   |
+      | {name: null, name2: null}           | name    | true   |
+      | {name: null, name2: null}           | name2   | true   |
```

Comment:
what is the difference between these two cases? Isn't one redundant?

Reply:
Well the second column is. Side-by-side, the two examples show, that it does not matter which field you test existence on or where this field in the order given by the literal.